src/main/resources/templates/db_scripts/populatePublicationPeriod.sql

```diff
@@ -28,19 +36,19 @@ CREATE OR REPLACE FUNCTION ${myuniversity}_${mymodule}.parse_publication_period(
   END;
 $$ LANGUAGE sql IMMUTABLE PARALLEL SAFE STRICT;
 
-ALTER TABLE ${myuniversity}_${mymodule}.instance DISABLE TRIGGER USER;
+ALTER TABLE instance DISABLE TRIGGER USER;
```

Comment:
If this script is going to be run whilst the system is operational and accessible, is it still appropriate to disable all of the triggers whilst it is being executed?

Reply:
No.

Reply:
That answer surprises me, I would have thought that disabling triggers could interfere with the integrity of the data in the system (presumably that's why they are there).